CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+CHANGES IN VERSION 1.16.0 (FROM 1.15.1)
+===================================
+
+Appium 1.16.0 is a minor release
+
+#### General
+
```

Comment:
#### General
* Fix handling of unexpected shutdown (https://github.com/appium/appium-base-driver/pull/370) (https://github.com/appium/appium/pull/13635) (https://github.com/appium/appium/pull/13662)
* Beta support for Flutter (https://github.com/appium/appium/pull/12945)

Reply:
#### IOS (General)
* Fix creation of simulators on Xcode patch versions (https://github.com/appium/node-simctl/pull/89)
* Support setting geolocation on simulators using [Lyft's set-simulator-location CLI](https://github.com/lyft/set-simulator-location) (https://github.com/appium/appium-ios-simulator/pull/249)